src/pages/organize/[orgId]/people/[personId]/index.tsx

```diff
@@ -56,10 +56,10 @@ const PersonProfilePage: PageWithLayout<PersonProfilePageProps> = ({
       </Head>
       <Grid container direction="row" spacing={6}>
         <Grid item lg={4}>
-          <PersonDetailsCard {...{ person }} />
+          <PersonDetailsCard {...person} />
         </Grid>
         <Grid item lg={4}>
-          <PersonOrganisationsCard {...{ orgId, personId }} />
+          <PersonOrganizationsCard {...props} />
```

Comment:
w.r.t to the "tie together" comment: Right now the props of these two components _are_ the same - if this changes later, then whomever makes the change can update the interface and passing of props if necessary. 

In terms of being explicit, the interface is clearly defined as PersonProfilePageProps 2 lines above the component. If the type def was moved to another file, then maybe it might just be worth typing out the props again, explicitly.

Reply:
Ok, my comments are a little bit all over the place, because I was confused by how the threads "move around" (or rather, don't move around) when the underlying code changes. I'm sorry for the confusion. But I hope I managed to get the general principle across, which is that we should not use API interfaces as proptypes, as that ties together interfaces between different "layers" of the code.